test/TesterAzureUtils/Streaming/HaloStreamSubscribeTests.cs

```diff
@@ -42,15 +42,13 @@ public void Configure(ISiloHostBuilder hostBuilder)
                         .AddMemoryGrainStorage("MemoryStore", options => options.NumStorageGrains = 1)
                         .AddAzureTableGrainStorage("AzureStore", builder => builder.Configure<IOptions<ClusterOptions>>((options, silo) =>
```

Comment:
if we're not reading from ClusterOptions we don't need to use the builder in all these cases. 
call should be something like:

    .AddAzureTableGrainStorage("AzureStore", options => 
    { 
        options.ConnectionString = TestDefaultConfiguration.DataConnectionString; 
         options.DeleteStateOnClear = true; 
    })

This applies to all of the AddAzureTableGrainStorage and AddAzureTableGrainStorageAsDefault calls which use the options builder to get information from the ClusterOptions